specs/src/specs/params.md

```diff
@@ -0,0 +1,50 @@
+# Celestia Governance Params
+
+These are the parameters for mainnet. Note that not all of these parameters are
+changable via governance. This list also includes parameter that require a
+hardfork to change due to being manually hardcoded in the application or they
+are blocked by the `x/paramfilter` module.
+
+## Parameters
+
+| Module.Parameter | Default | Summary | Changeable via Governance |
+| --- | --- | --- | --- |
+| auth.MaxMemoCharacters | 256 | Largest allowed size for a memo in bytes. | True |
+| auth.TxSigLimit | 7 | Max number of signatures allowed in a multisig transaction. | True |
+| auth.TxSizeCostPerByte | 10 | Gas used per transaction byte. | True |
+| auth.SigVerifyCostED25519 | 590 | Gas used to verify Ed25519 signature. | True |
+| auth.SigVerifyCostSecp256k1 | 1000 | Gas used to verify secp256k1 signature. | True |
+| bank.SendEnabled | true | Allow transfers. | False |
+| blob.GasPerBlobByte | 8 | Gas used per blob byte. | True |
+| MaxBlockBytes | 100MiB | Hardcoded value in CometBFT for the protobuf encoded block. | False |
+| MaxSquareSize | 128 | Hardcoded value in the applications that requires a hardfork to change. | False |
+| blob.GovMaxSquareSize | 64 | Governance parameter for the  square size. If larger than MaxSquareSize, MaxSquareSize is used. | True |
+| consensus.block.MaxBytes | 1.88MiB | Governance parameter for the maximum size of the block. | True |
+| consensus.block.MaxGas | -1 | Maximum gas allowed per block (-1 is infinite). | True |
+| consensus.block.TimeIotaMs | 1000 | Minimum time added to the time in the header each block. | False |
+| consensus.evidence.MaxAgeNumBlocks | 100000 | The maximum number of blocks before evidence is considered invalid. This value will stop CometBFT from pruning block data. | True |
+| consensus.evidence.MaxAgeDuration | 2 days | The maximum age of evidence before it is considered invalid. This value should be identical to the unbonding period. | True |
```

Comment:
From this it remains unclear in which actual unit this param would be defined (IIRC it's seconds not days?).

Reply:
```suggestion
| consensus.evidence.MaxAgeDuration | 172800000000000 (2 days) | The maximum age (in nanoseconds) of evidence before it is considered invalid. This value should be identical to the unbonding period. | True |
```

Reply:
its defined a golang time.Duration which is always nanoseconds, so I just listed two days. If encoded properly, it can be represented as seconds I believe (like in other sdk modules), but since these are comet params they are not encoded normally.

will change to nano seconds

Reply:
this was also updated to what it will be for mainnet (21 days same as unbonding) in nanoseconds. The default will reflect that after #2417 and https://github.com/celestiaorg/cosmos-sdk/pull/345